boss-ws/src/main/java/org/genomebridge/boss/http/resources/FCSResource.java

```diff
@@ -2,25 +2,33 @@
 
 import java.util.concurrent.ConcurrentHashMap;
 
-import javax.ws.rs.DELETE;
-import javax.ws.rs.GET;
-import javax.ws.rs.HEAD;
-import javax.ws.rs.HeaderParam;
-import javax.ws.rs.PUT;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
+import javax.ws.rs.*;
```

Comment:
This is opinion, not a must-do on the pull request, but I very much prefer having the 7 imports rather than a *.  The problem is that for very common names (like GET), there are often dozens of places where they are defined.  Without the explicit import it can be hard to know which one you're getting.  IDE's usually sort this out nicely, but when I'm looking at code outside of an IDE, it's obscure.  We ought to have a discussion, a vote, and a coding standard, I guess.